src/protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,208 @@
+
+```console
+$ curl https://localhost:5001/me -H "Authorization: Bearer {valid-access-token}"
+Graph API call result:
+{
+   "version": "1.1",
+   "content": {
+      "headers": [
+         {
+            "key": "Content-Type",
+            "value": [
+               "application/json; odata.metadata=minimal; odata.streaming=true; IEEE754Compatible=false; charset=utf-8"
+            ]
+         }
+      ]
+   },
+   "statusCode": 200,
+   "reasonPhrase": "OK",
+   "headers": [
+     ...
+   ],
+   "trailingHeaders": [],
+   "requestMessage": {
+      "version": "1.1",
+      "versionPolicy": 0,
+      "content": null,
+      "method": {
+         "method": "GET"
+      },
+      "requestUri": "https://graph.microsoft.com/v1.0/me/",
+      "headers": [
+        ...
+      ],
+      "properties": {},
+      "options": {}
+   },
+   "isSuccessStatusCode": true
+}
+```
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+- An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Setup
+
+### 1. Register the web API application in your Azure Active Directory (Azure AD)
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                           | Notes                                                                                                       |
+|:------------------------------:|:--------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-protected-api-access-protected-api`               | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**  | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | **Web**                                                             | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
```

Comment:
Why does this need a platform type?  An API typically wouldn't as it's not redirecting users anywhere.

Reply:
thanks for the feedback @ckittel, copy pasta

done  | addressed from [9f9b8](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/13/commits/9f9b8bc6efe81bff70f00c1676c5b6036cdbe1b3)

Reply:
No more platform type.

Also, the "Expose an API" section seems to look align with other approved examples.